deltacat/compute/compactor_v2/utils/task_options.py

```diff
@@ -49,10 +49,7 @@ def _calculate_parquet_column_size(
             if column_meta.path_in_schema in columns:
                 columns_found += 1
                 column_size += column_meta.total_uncompressed_size
-        assert columns_found == len(columns), (
-            "Columns not found in the parquet data as "
-            f"{columns_found} != {len(columns)}"
-        )
+        # NOTE: as delete deltas may only contain a subset of columns we should no longer assert that assert columns_found == len(columns)
```

Comment:
I don't think this assertion will be invalid if we propagate right set of columns which is my another comment above.